src/group_theory/coset.lean

```diff
@@ -303,6 +303,21 @@ quotient_group.eq
 lemma out_eq' (a : quotient s) : mk a.out' = a :=
 quotient.out_eq' a
 
+variables (s)
+
+/- It can be useful to write `obtain ⟨h, H⟩ := mk_out'_eq_mul ...`, and then `rw [H]` or
+  `simp_rw [H]` or `simp only [H]`. In order for `simp_rw` and `simp only` to work, this lemma is
+  stated in terms of an arbitrary `h : s`, rathern that the specific `h = g⁻¹ * (mk g).out'`. -/
+@[to_additive quotient_add_group.mk_out'_eq_mul]
+lemma mk_out'_eq_mul (g : α) : ∃ h : s, (mk g : quotient s).out' = g * h :=
```

Comment:
You now don't actually need this lemma, right?

Reply:
I think so.